permissions/plugin-atlassian-bitbucket-server-integration.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "atlassian-bitbucket-server-integration"
+github: "jenkinsci/atlassian-bitbucket-server-integration-plugin"
+paths:
+- "io/jenkins/plugins/atlassian-bitbucket-server-integration"
+developers:
+- "mhenschke_atlassian"
```

Comment:
Maybe we could add a few more people here? I say the entire team to reduce the "bus factor"

Reply:
Are you happy with the three I've added, or should I chase up the rest of the team to create accounts?